ansible/group_vars/all

```diff
@@ -34,10 +34,6 @@ runtimesManifest:
   defaultImageTag: "latest"
   runtimes:
     nodejs:
-    - kind: "nodejs"
```

Comment:
@rabbah There are errors in Manifest tests cases, when I remove this kinds, it means I need to adjust the test cases?
```
 Exec should initialize exec manifest FAILED
    java.util.NoSuchElementException: None.get
```